github/REVIEWING.md

```diff
@@ -0,0 +1,341 @@
+
+1. Make sure the lines in the body are limited to 72 characters.
+
+1. Make sure there are no typos or spelling mistakes in the title or body.
+
+1. Make sure the author is properly stated, generally in the form of `Firstname Lastname <email>`.
+
+1. If the commit conventions use or imply using a [`Signed-off-by` line](https://stackoverflow.com/a/14044024/4804196), make sure that line also has the author properly stated.
+   Also make sure the exact same name are used for both the author and the `Signed-off-by` line.
+
+1. Make sure the commit message is readable and to the point.
+   Make sure it clearly details what the commit does.
+   Make sure it fits the content of the commit (the actual contribution);
+   i.e. when you check the changes in the commit, they are well connected to the message;
+   that is they don't mention more or less than the commit contents.
+
+You will submit any issues in the general review message.
+
+## Review Contents
+
+Reviewing contents is done both in the pull request GitHub interface and locally.
+
+In the pull request GitHub interface, go the to the `Files changed` tab where you can see an interactive view of the changes.
+Walk through the changes and check for anything that may not be good / correct:
+
+- hard to read content
+- typos
+- violation of general coding / writing conventions
+  - lines to large
+  - missing newlines at the end of the files
+  - training white spaces
+  - improper indentation
+- violation of project coding / writing conventions
+- general coding / writing mistakes
+- project-specific coding / writing mistakes
+
+Follow the [GitHub review guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and add your request for changes.
+In the GitHub interactive interface in the `Files changed` tab you can request changes either on a single line or on multiple lines, where the `+` icon appears when hovering.
+You can request changes as feedback text or as concrete content update suggestions, as shown in the [GitHub review guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and add your request for changes.
+
+Locally you can get a unified view of the changes inside files.
+You can open a file and check out its entirety: existing content and newly contributed content.
+You can check the newly contributed content and make sure that:
+
+- It is consistent with existing content: same structure, same format, same conventions.
+- It is readable and integrates well with existing content.
+
+Finally, you want to be sure that the content is well aligned with the commits it is part of.
+Each commit should do one thing and do one thing well, and the content and commit should be aligned.
```

Comment:
This assumes that the repo reviews each commit individually, but there are other repos that review at PR level, squashing all commits upon merging the PR.